test/setupTests.ts

```diff
@@ -3,3 +3,9 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import "@testing-library/jest-dom/extend-expect";
+
+process.env.AAD_CLIENT_ID = "test_client_id";
+process.env.AAD_CLIENT_SECRET = "test_client_secret";
+process.env.AAD_TENANT_ID = "test_tenant_id";
+process.env.AAD_API_ID = "test_api_id";
+process.env.API_URL = "http://localhost:8080/spring-api";
```

Comment:
Create a consistent set of environment variables used only in Jest's test environment.  See: https://stackoverflow.com/a/58953365